action/install.go

```diff
@@ -86,44 +92,27 @@ func isSamePath(src, dst string) (bool, error) {
 
 // uploadManifests sends manifests to Kubectl in a particular order.
 func uploadManifests(c *chart.Chart, namespace string, dryRun bool) error {
-	// The ordering is significant.
-	// TODO: Right now, we force version v1. We could probably make this more
-	// flexible if there is a use case.
-	for _, o := range c.Namespaces {
-		if err := marshalAndCreate(o, namespace, dryRun); err != nil {
-			return err
-		}
-	}
-	for _, o := range c.Secrets {
-		if err := marshalAndCreate(o, namespace, dryRun); err != nil {
-			return err
-		}
-	}
-	for _, o := range c.PersistentVolumes {
-		if err := marshalAndCreate(o, namespace, dryRun); err != nil {
-			return err
-		}
-	}
-	for _, o := range c.ServiceAccounts {
-		if err := marshalAndCreate(o, namespace, dryRun); err != nil {
-			return err
-		}
-	}
-	for _, o := range c.Services {
-		if err := marshalAndCreate(o, namespace, dryRun); err != nil {
-			return err
-		}
-	}
-	for _, o := range c.Pods {
-		if err := marshalAndCreate(o, namespace, dryRun); err != nil {
-			return err
+
+	log.Warn("Matt forgot to write the code to modify the versioned object.")
```

Comment:
Yeah, this is still a work in progress, and I have some stuff in there to shame me into remembering to do things. ;-)